docs/elements.md

```diff
@@ -171,12 +171,15 @@ Attribute | Type | Default | Description
 `max-correct` | integer | special | The maximum number of correct answers to display. Defaults to displaying all correct answers.
 `fixed-order` | boolean | false | Disable the randomization of answer order.
 `partial-credit` | boolean | false | Enable partial credit scores. By default, the choice of grading method is "all-or-nothing".
-`partial-credit-method` | string | 'PC' | Two grading methods for partial credit: 'EDC' (Every Decision Counts) and 'PC' (Percent Correct). See explanation below.
-`hide-help-text` | boolean | false | Help text with hint regarding the selection of answers. Popover button describes the selected grading algorithm ('all-or-nothing', 'EDC' or 'PC')
+`partial-credit-method` | string | 'PC' | Three grading methods for partial credit: 'COV' (Coverage), 'EDC' (Every Decision Counts), and 'PC' (Percent Correct). See explanation below.
+`hide-help-text` | boolean | false | Help text with hint regarding the selection of answers. Popover button describes the selected grading algorithm ('all-or-nothing', 'COV', 'EDC' or 'PC')
 `detailed-help-text` | boolean | false | Display detailed information in help text about the number of options to choose.
 `hide-answer-panel` | boolean | false | Option to not display the correct answer in the correct panel.
 `hide-letter-keys` | boolean | false | Hide the letter keys in the answer list, i.e., (a), (b), (c), etc.
 `hide-score-badge` | boolean | false | Hide badges next to selected answers.
+`min-select` | integer | special | The minimum number of answers that must be selected in any valid submission.
+`max-select` | integer | special | The maximum number of answers that can be selected in any valid submission.
+`show-number-correct` | boolean | false | Display the number of correct choices in the help text.
```

Comment:
What happens if we select both `show-number-correct` and `detailed-help-text`? Could we add some docs to describe this? Or maybe we should add some checks in the code to only allow one of them to be selected at a time?

Reply:
They are independent of each other. Setting `show-number-correct` tells you how many correct options there are -- it adds the text "There are exactly X correct options in the list above" to the help text. On the other hand, `detailed-help-text` tells you how many options you have to select -- it adds the text "Select between Y and Z options" to the help text. Therefore, if both are specified, then the help text looks like this: "Select between Y and Z options. There are exactly X correct options."

Reply:
I think the fix here is to clarify the documentation for `detailed-help-text`.

Reply:
I should also add that in some situations, setting `detailed-help-text="true"` has no effect. This happens if `min/max-select` are set to some non-default value -- in that situation, we always tell students to "Select between `min-select` and `max-select` options" no matter whether `detailed-help-text` is specified or not.

Reply:
Documentation for `detailed-help-text` has been added in my latest commit.